R/g.impute.R

```diff
@@ -233,7 +233,12 @@ g.impute = function(M, I, params_cleaning = c(), desiredtz = "",
       for (j in 1:ndays) { 
         missing = which(is.na(imp[,j]) == T)
         if (length(missing) > 0) {
-          imp[missing,j] = imp3[missing]
+          if (part2ImpWithZero) { # If part2ImpWithZero = TRUE, Only advisable when the accelerometer has not been worn during sleep.
+            if (colnames(metashort)[mi] != "EN") imp[missing,j] = 0
```

Comment:
How about the angle metrics. I think they should also be ignored for the imputation, because 3 time zero for the three angles is not a plausible imputation. Instead, maybe just skip angle metrics just like you skip EN.
Similarly, we may need to skip the rolling median metrics as those can neither be imputed by zero.

Reply:
Done, angles and rolling median metrics are now ignored here.